lib/util/get-string-from-node.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @fileoverview Utility for retrieving the final translatable string from an AST
+ * node for tests that focus on the strings themselves.
+ * @author Automattic
+ * @copyright 2016 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+/**
+ * Returns the singular and maybe plural strings from a CallExpression.
+ *
+ * @param  {Object} node CallExpression node
+ * @return {Object}      First non-sequence callee
+ */
+var getStringLeafNodes = require( './get-string-leaf-nodes' );
+
+function getStringFromNode( node ) {
```

Comment:
I had a look at reconstructing the string, and it's a bit interesting, but then I realized that any expression will foul up the translation system, so for the purposes of this module, you can assume there will never be more than one quasi.

This definitely warrants a note in the docstring.